types/store.go

```diff
@@ -222,3 +230,28 @@ func (key *KVStoreKey) Name() string {
 func (key *KVStoreKey) String() string {
 	return fmt.Sprintf("KVStoreKey{%p, %s}", key, key.name)
 }
+
+// TODO: Move to TmLibs
```

Comment:
What other repos will want to use this to justify moving to tmlibs?

Reply:
Possibly github.com/tendermint/iavl

Reply:
sure - newly, I've become a fan of keeping everything in the origin library until the use case specifically comes up to move to the tmlibs library - keeps that library clean (aka rational is not in there, but maybe eventually one day will be)

I'd propose that we add a section in tmlibs which accounts/outlines possible functions which could be moved need be... I'll open up an issue